Source/MSBuildTools.Unity/Packages/com.microsoft.msbuildforunity/Editor/ProjectGenerator/Scripts/MSBuildTools.cs

```diff
@@ -382,7 +382,7 @@ private static void RegenerateEverything(UnityProjectInfo unityProjectInfo, bool
 
                 foreach (string otherFile in TemplateFiles.Instance.OtherFiles)
                 {
-                    File.Copy(otherFile, Path.Combine(Utilities.MSBuildProjectFolder, Path.GetFileName(otherFile)));
+                    File.Copy(otherFile, Path.Combine(Utilities.AssetPath, Path.GetFileName(otherFile)));
```

Comment:
I think you may want to undo this, as we want to copy "otherfiles" from templates to the MSBuildProjectFolder.

What you want to do, is above the ForEach, copy the NuGet.config template to Utilities.AssetPath

Reply:
Fixed